### PR TITLE
Added info about AlmaLinux Secure Boot to Security Measures

### PR DESCRIPTION
**Pull request by sboldyreva**

### CODE REVIEW

**Comment by sboldyreva** [comment]
@bennyvasquez @andrewlukoshko @codyro @jonathanspw

File: layouts/security/single.html [hunk 1]
```diff
@@ -167,7 +167,41 @@ <h2 class="pb-2 text-center border-bottom" style="border-bottom-color: #14598a!i
...
           </div>
-        </div>       
+        </div>  
+        <div class="SecureBoot-container" style="display: flex; flex-direction: column; padding: 40px 20px;">
+          <h2 class="pb-2 text-center border-bottom" style="border-bottom-color: #14598a!important;">Secure Boot</h2>
+          <div class="al-article-content pb-3 al-wysiwyg" style="padding: 0px!important;">
+            <section>
+              <p>
+                {{ i18n "Secure Boot is a security feature that ensures a system boots only with trusted software, preventing the loading of unauthorized operating systems or software that could potentially compromise the system's integrity." }} 
+                <br>{{ i18n "AlmaLinux provides Secure Boot support starting the AlmaLinux 8.4 release." }}
```
Comment (bennyvasquez) [inline layouts/security/single.html:177]:
```suggestion
                <br>{{ i18n "AlmaLinux provides Secure Boot support starting with the AlmaLinux 8.4 release." }} 
```

**Review by bennyvasquez — COMMENTED** [review]
Only one small grammar suggestion here. I'd love to see a blog post about why secure boot is important and how a user might use it, too!

**Comment by sboldyreva** [comment]
@bennyvasquez thanks for the catch! Updated the i18n en file

**Review by codyro — APPROVED** [review]

**Comment by andrewlukoshko** [comment]
> I'd love to see a blog post about why secure boot is important and how a user might use it, too!

It's something usually enabled by default and user doesn't have to do anything to use it. They can only disable it if necessary.
But we can make basic recommendations still how to check if it's enabled to make sure everything is fine.
Secure Boot support is not AlmaLinux exclusive feature just in case.

**Comment by bennyvasquez** [comment]
@andrewlukoshko for sure! It just allows us to highlight the fact that we support it outside of just adding to our documentation.

**Pull request merged**